src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1114,5 +1119,282 @@ public void SarifTransformerTests_ToCurrent_BasicResult()
 
             v2LogText.Should().Be(v2LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_TwoResultsWithFixes()
+        {
+            string v1LogText =
+              @"{
+                  ""version"": ""1.0.0"",
+                  ""runs"": [
+                    {
+                      ""tool"": {
+                        ""name"": ""CodeScanner"",
+                        ""semanticVersion"": ""2.1.0""
+                      },
+                      ""files"": {
+                        ""http://localhost:34420/HtmlFixes.html"": {
+                          ""mimeType"": ""text/html""
+                        }
+                      },
+                      ""results"": [
+                        {
+                          ""ruleId"": ""WEB1079"",
+                          ""formattedRuleMessage"": {
+                            ""formatId"": ""default"",
+                            ""arguments"": [
+                              ""shape""
+                            ]
+                          },
+                          ""suppressionStates"": [ ""suppressedExternally"" ],
+                          ""baselineState"": ""existing"",
+                          ""level"": ""error"",
+                          ""snippet"": ""<area alt=\""Here is some text\"" coords=\""10 20 20\"" href=\""moon.html\"" shape=circle xweb:fixindex=\""0\"" />"",
```

Comment:
For the sake of brevity, I suggest that the example log files include only those properties that are either (a) required, (b) necessary to demonstrate the feature, or (c) helpful for understanding. In this example, `suppressionState`, `baselineState`, and `level` are optional, so I'd omit them, `snippet` helps clarify the example, so I'd keep it even though it's optional.

Other candidates for omission:
- `tool.semanticVersion` (used to be required, but is now optional)
- `run.files`

Reply:
Done